specs/cli/gencli.jl

```diff
@@ -15,7 +13,7 @@ CommandLine execution context.
 `pipelineopts`: keyword arguments that should be used to further customize the `pipeline` creation
 \"\"\"
 Base.@kwdef struct CommandLine
-    exec::Base.Function = OpenPolicyAgent_jll.opa
+    exec::Base.Function = ()->`opa`
```

Comment:
This needs an update maybe? It seems to be inconsistent with `cli.jl`?

Reply:
Thanks for catching. Done in https://github.com/JuliaComputing/OpenPolicyAgent.jl/pull/17/commits/55c894db20ee7abc2c4cb8e222473ee743095a17